src/ExpressionEvaluator/Core/Source/ResultProvider/Formatter.cs

```diff
@@ -188,6 +219,8 @@ string IDkmClrFullNameProvider.GetClrExpressionForThis(DkmInspectionContext insp
 
         internal abstract bool IsWhitespace(char c);
 
+        internal abstract string SimplifyCompilerGeneratedName(string name);
```

Comment:
>SimplifyCompilerGeneratedName [](http://example.com/codeflow?start=33&length=29)

I think this would be better: `GetOriginalName(string compilerGeneratedName)`